bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/serial_bridge.xml

```diff
@@ -4,12 +4,12 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
-	<bridge-type id="pentair_serial_bridge">
-		<label>Serial Bridge</label>
+	<bridge-type id="serial_bridge">
+		<label>Pentair-RS485 Serial Bridge</label>
```

Comment:
Are you sure you want to rename the Thing Type ID? This won't be backward compatible. Users will have to modify their configurations when updating openHAB.

Reply:
I am just trying to be consistent in naming the bridges (they should either both have pentair_ or neither should).  Unfortunately, since I was not able to include these improvements into OH2.x, I hav already been distributing the improved binding as .jar to many folks which included these changes.  I will include special mention of this in the README file.

Reply:
OK